plugin/src/main/java/org/openrewrite/gradle/RewriteExtension.java

```diff
@@ -33,7 +33,7 @@
     private final Project project;
     private File configFile;
     private String metricsUri = magicalMetricsLogString;
-    private String rewriteVersion = "7.2.2";
+    private String rewriteVersion = "7.3.0";
```

Comment:
@sambsnyd just to double-check, this is technically all that should be necessary to "bump the version" of `rewrite` core used to 7.3.0?

Reply:
going to guess that the exact same time review approval means "yes" ; )